ui/page/components/wallet_setup_page.go

```diff
@@ -114,11 +114,9 @@ func NewCreateWallet(l *load.Load, walletCreationSuccessCallback func(), assetTy
 
 	pg.passwordEditor = l.Theme.EditorPassword(new(widget.Editor), values.String(values.StrSpendingPassword))
 	pg.passwordEditor.Editor.SingleLine, pg.passwordEditor.Editor.Submit = true, true
-	pg.passwordEditor.Hint = values.String(values.StrSpendingPassword)
 
-	pg.confirmPasswordEditor = l.Theme.EditorPassword(new(widget.Editor), values.String(values.StrSpendingPassword))
+	pg.confirmPasswordEditor = l.Theme.EditorPassword(new(widget.Editor), values.String(values.StrConfirmSpendingPassword))
 	pg.confirmPasswordEditor.Editor.SingleLine, pg.confirmPasswordEditor.Editor.Submit = true, true
-	pg.confirmPasswordEditor.Hint = values.String(values.StrConfirmSpendingPassword)
```

Comment:
Why delete the hint?

Reply:
hit include when init Editor in l.Theme.EditorPassword()